crates/biome_css_parser/src/parser.rs

```diff
@@ -44,10 +48,21 @@ impl CssParserOptions {
         self
     }
 
+    /// Enables parsing of Grit metavariables.
+    pub fn allow_grit_metavariable(mut self) -> Self {
```

Comment:
nit: maybe we should use the plural form `allow_grit_metavariables` for consistency.